src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -52,7 +62,9 @@ type Props = {
 type State = {
     isInputOpen: boolean,
     approverSelectorContacts: Array<User>,
-    mentionSelectorContacts: Array<User>
+    mentionSelectorContacts: Array<User>,
+    feedItems: Array<Comment | Task | BoxItemVersion>,
+    pendingItems: Array<Object>
```

Comment:
AFAIK I can remove pending items, since we've decided to wait until we have all three item types for the activity feed, before propagating `feedItems`

Reply:
+1 to removing